ui/src/app/core/services/project.service.ts

```diff
@@ -6,8 +7,8 @@ import { Subject } from 'rxjs';
   providedIn: 'root',
 })
 export class ProjectService {
-  baseUrl = 'http://localhost:3000/project';
-  projects: Subject<any[]> = new Subject<any[]>();
+  baseUrl = 'http://localhost:3000/api/project';
```

Comment:
project should have global baseUrl. Which should configurable based on environment. In your local baseUrl is localhost:3000, In other environment it will change. Check any service in turnkey. For example in label service:
 baseUrl = environment.coreBaseUrl + '/api/label';    coreBaseUrl is read from environment file. Those are called environment variables.We need exactly same implementation